docs/dev/framework/hooks.md

```diff
@@ -131,6 +131,8 @@ legacy registered hooks.
 {{< version "4.8" >}}
 
 Contao also supports its own annotation formats via the [Service Annotation Bundle][ServiceAnnotationBundle].
+This means that it is only necessary to create one file for the respective adaptation. It should be noted,
+however, that one has to adhere to the provided [Contao folder structure][ContaoFolderStructure].
```

Comment:
@zonky2 this applies to everything, not just hooks. Also using DCA files in `src/Resources/contao`, like you did, should not cause any troubles with the service auto registration of the Contao Manager bundle. Something else must have been the issue.

Reply:
DCA files in `src/Resources/contao` works fine!